acceptance/tests/peering/peering_connect_test.go

```diff
@@ -287,6 +296,70 @@ func TestPeering_Connect(t *testing.T) {
 				k8s.CheckStaticServerConnectionSuccessful(t, staticClientOpts, staticClientName, "http://localhost:1234")
 			}
 
+			// Check that requests can reach the external static-server from the static client cluster.
+			if cfg.EnableTransparentProxy {
+				const (
+					externalServerK8sNamespace = "external"
+					externalServerServiceName  = "static-server.external"
+					externalServerHostnameID   = "static-server-hostname"
+					externalServerIPID         = "static-server-ip"
+					terminatingGatewayRules    = `service_prefix "static-server" {policy = "write"}`
+				)
+				externalServerOpts := &terratestk8s.KubectlOptions{
+					ContextName: staticServerOpts.ContextName,
+					ConfigPath:  staticServerOpts.ConfigPath,
+					Namespace:   externalServerK8sNamespace,
+				}
+
+				logger.Logf(t, "creating namespace %s in server peer", externalServerK8sNamespace)
+				k8s.RunKubectl(t, staticServerPeerClusterContext.KubectlOptions(t), "create", "ns", externalServerK8sNamespace)
+				helpers.Cleanup(t, cfg.NoCleanupOnFailure, cfg.NoCleanup, func() {
+					k8s.RunKubectl(t, staticServerPeerClusterContext.KubectlOptions(t), "delete", "ns", externalServerK8sNamespace)
+				})
+
+				// Create the external server in the server Kubernetes cluster, outside the mesh in the "external" namespace
+				logger.Log(t, "creating static-server deployment in server peer outside of mesh")
+				k8s.DeployKustomize(t, externalServerOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/bases/static-server-https")
+
+				if c.ACLsEnabled {
+					terminatinggateway.UpdateTerminatingGatewayRole(t, staticServerPeerClient, terminatingGatewayRules)
+				}
+
+				// Prevent dialing the server directly through the sidecar.
+				terminatinggateway.CreateMeshConfigEntry(t, staticServerPeerClient, "")
+
+				// Create the config entry for the terminating gateway
+				terminatinggateway.CreateTerminatingGatewayConfigEntry(t, staticServerPeerClient, "", "", externalServerHostnameID, externalServerIPID)
+
+				externalServerIP, err := k8s.RunKubectlAndGetOutputE(t, staticServerPeerClusterContext.KubectlOptions(t), "get", "-n", "external", "pods", "-l", "app=static-server", "-o=jsonpath={.items[0].status.podIP}")
+				require.NoError(t, err)
+				require.NotEmpty(t, externalServerIP)
+
+				externalServerHostnameURL := fmt.Sprintf("https://%s.virtual.%s.consul", externalServerServiceName, staticServerPeer)
+				externalServerIPURL := fmt.Sprintf("http://%s", externalServerIP)
+				fmt.Println(externalServerIPURL)
+
+				helpers.RegisterExternalService(t, staticServerPeerClient, "", externalServerServiceName)
+
+				terminatinggateway.CreateServiceDefaultDestination(t, staticServerPeerClient, "", externalServerHostnameID, "http", 443, externalServerServiceName)
```

Comment:
I'm not sure this case is supported because you've defined the service protocol as `http`, but when you use TLS at the client all the sidecar sees is encrypted traffic. Looking at the table here for reference: https://developer.hashicorp.com/consul/docs/k8s/connect/terminating-gateways#register-external-services-with-consul-1